python/lvmdrp/functions/rssMethod.py

```diff
@@ -2724,7 +2724,6 @@ def DAR_registerSDSS_drp(
         plt.show()
 
 
-@drop_missing_input_paths(["in_rss"])
```

Comment:
Why you had to remove this?

Reply:
This causes the function to crash when the input list is missing a channel.  For example, when combining r, z, and b is missing, like for 60010, when the input was [[], [r_file], [z_file].   The function handles cases where one or more of the channels are None, so it can still combine the rest and produce an output.  

We can add it back if you want to also check for missing files but it needs to be tweaked to handle the above case.

Reply:
Ok I see. I did not account for Nones in `in_rss`. I'll fix this on a separated branch.